chpl/chpl-service/src/main/java/gov/healthit/chpl/entity/FilesEntity.java

```diff
@@ -0,0 +1,145 @@
+package gov.healthit.chpl.entity;
+
+import java.util.Date;
+
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "files")
+public class FilesEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Basic(optional = false)
+    @Column(name = "file_id", nullable = false)
+    private Long id;
+
+    @Basic(optional = false)
+    @OneToOne(optional = false, fetch = FetchType.LAZY)
+    @JoinColumn(name = "file_type_id", nullable = false, insertable = true,
+            updatable = false)
+    private FileTypeEntity fileType;
```

Comment:
Do you also need a mapping to file_type_id column in here that's just the mapping to the Long column value and is insertable and updatable? I think we normally have that but maybe it's not needed?